site/multiembedder/MultiEmbedder.tsx

```diff
@@ -200,6 +232,29 @@ class MultiEmbedder {
     embedAll() {
         this.addFiguresFromDOM().watchScroll()
     }
+
+    setUpGlobalEntitySelectorForEmbeds() {
+        const element = globalEntitySelectorSelector()
+        if (!element) return
+
+        const selectionParam = getWindowQueryParams().selection
+        const defaultCountry = element.getAttribute(
+            GLOBAL_ENTITY_SELECTOR_DEFAULT_COUNTRY
+        )
```

Comment:
We should probably have some way to pass _multiple_ countries as the default selection, no?
[Charlie requested this just last week.](https://owid.slack.com/archives/C46U9LXRR/p1612279381009700) (related commit on `master`: https://github.com/owid/owid-grapher/commit/c9916e2bf36ee40b6696384b5214f48b9a2fa558)

Reply:
done